client/my-sites/customer-home/free-photo-library-card/index.jsx

```diff
@@ -23,71 +23,49 @@ import './style.scss';
 /**
  * Image dependencies
  */
-import freePhotoLibraryVideoPrompt from 'assets/images/customer-home/free-photo-library-video-prompt.png';
+import freePhotoLibraryVideoPrompt from 'assets/images/customer-home/illustration--free-photo-library.svg';
 
 const FreePhotoLibraryCard = ( {
 	recordTracksEvent: tracks,
 	openSupportArticleDialog: supportArticleDialog,
 } ) => {
-	const [ showDialog, setShowDialog ] = useState( false );
 	const translate = useTranslate();
 
-	const toggleDialog = () => {
-		if ( ! showDialog ) {
+	const displayDialog = clickSource => {
+		if ( clickSource === 'prompt' ) {
 			tracks( 'calypso_customer_home_free_photo_library_video_dialog_view' );
+		} else {
+			tracks( 'calypso_customer_home_free_photo_library_video_support_page_view' );
```

Comment:
Non-blocking, but I noticed while refactoring quick links that we're also bumping stats under the 'calypso_customer_home' link. Might be worth adding something like this this
```jsx
composeAnalytics(
    bumpStat( 'calypso_customer_home', 'view_free_photo_library_video' ),
    tracks( 'calypso_customer_home_free_photo_library_video_support_page_view' ),
)
```

Reply:
ff69cfc1f14fb9a72e1a743dadd1d14444cf3574